### ISSUE TITLE
close (x) button needs to be clicked twice after swipe action on mobile

### ISSUE THREAD

**Issue by vesper8** [RobinCK/vue-gallery#109]
I tried your own demo on mobile just now. Arrows work well, swipe works well. I noticed if you click on an image and go into the Lightbox carousel and immediately click the X to close, it works as intended, if however you do the same but perform a swipe action at least once, then clicking the X button does nothing, you have to click it a 2nd time before the Lightbox will close

**Comment by consuman** [comment]
Yeah, I noticed the same problem! Would be great if it would be fixed. Gallery is pretty awsome :) But this would be bad for productive enviroment.

**Comment by vesper8** [comment]
I'm handling it with jquery for now.. still hoping it gets fixed at some point

```
    $(document).on('click', '#blueimp-gallery a.close', () => {
      if (this.galleryImageIndex !== null) {
        this.galleryImageIndex = null;
      }
    });
```
